code/modules/admin/verbs/adminhelp.dm

```diff
@@ -387,23 +411,23 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 /datum/admin_help/proc/TicketPanel()
 	var/list/dat = list("<html><head><title>Ticket #[id]</title></head>")
 	var/ref_src = "[REF(src)]"
-	dat += "<h4>Admin Help Ticket #[id]: [LinkedReplyName(ref_src)]</h4>"
+	dat += "<h4>Ticket #[id]: [LinkedReplyName(ref_src)]</h4>"
 	dat += "<b>State: "
 	switch(state)
 		if(AHELP_ACTIVE)
-			dat += "<font color='red'>OPEN</font>"
+			dat += "<font color='#ff8c8c'>OPEN</font>"
 		if(AHELP_RESOLVED)
-			dat += "<font color='green'>RESOLVED</font>"
+			dat += "<font color='#9adb92'>RESOLVED</font>"

```

Comment:
Is the color difference so important we should sacrifice readability for it?

Reply:
You either sacrifice readability here or in the UI. Given how ubiquitous hex colors are, I don't honestly think this is a problem.